py-polars/tests/test_series.py

```diff
@@ -1640,6 +1639,16 @@ def test_ewm_mean_leading_nulls() -> None:
     ).to_list() == [None, None, 1.0, 1.0]
 
 
+@pytest.mark.parametrize(
+    "min_periods,expected", [(1, [1.0, 1.0, 1.0, 1.0]), (2, [None, None, None, None])]
+)
+def test_ewm_mean_min_periods(min_periods: int, expected: list[float | None]) -> None:
```

Comment:
Can we also add some tests that test for the numeric values with a bit more interesting data?